files/es/mdn/writing_guidelines/page_structures/macros/other/index.md

```diff
@@ -37,5 +37,4 @@ Estas macros han sido reemplazadas por otras formas de hacer lo mismo y ya no de
 
 ### Enlaces
 
-- La macro [`SectionOnPage`](https://github.com/mdn/yari/blob/main/kumascript/macros/SectionOnPage.ejs) crea una frase que enlaza con el nombre de una sección y el artículo que contiene esa sección. Por ejemplo, `\{{SectionOnPage("/es/docs/Mozilla/Firefox/Releases/21", "Cambios para desarrolladores web")}}` genera lo siguiente: _{{SectionOnPage("/es/docs/ Mozilla/Firefox/Releases/21", "Cambios para desarrolladores web")}}_.
 - La macro [`Link`](https://github.com/mdn/yari/blob/main/kumascript/macros/Link.ejs) inserta un enlace a la página especificada en MDN, utilizando el título de la página como la cadena visible para hacer clic y la información sobre herramientas extraída del resumen de SEO de la página.
```

Comment:
Hi @Graywolf9, we can also remove this macro mention. As we have removed this macro from [mdn/yari](https://github.com/mdn/yari/blob/main/kumascript/macros). See also: english document

![image](https://user-images.githubusercontent.com/15844309/213330769-9e30682c-f125-4c58-a16e-a27b8e5a2d61.png)

Reply:
Thank you so much! I didn't notice that